java/vector/src/main/java/org/apache/arrow/vector/file/ArrowWriter.java

```diff
@@ -64,7 +67,7 @@ public void writeRecordBatch(ArrowRecordBatch recordBatch) throws IOException {
   private void checkStarted() throws IOException {
     if (!started) {
       started = true;
-      start();
+      writeMagic();
```

Comment:
the schema is written in the footer - as far as i can tell, there was no reason to write it out a second time, and removing it didn't break anything.